osmaxx-py/osmaxx/excerptexport/forms/excerpt_export_form.py

```diff
@@ -0,0 +1,234 @@
+        label=_('East'),
+        required=True,
+    )
+    south = forms.FloatField(
+        label=_('South'),
+        required=True,
+    )
+
+    def form_layout(self):
+        return form_layout.Div(
+            form_layout.HTML('<h2>' + _('New excerpt') + '</h2>'),
+            form_layout.Fieldset(
+                _('Create new excerpt'),
+                'name',
+            ),
+            form_layout.Div(
+                form_layout.Fieldset(
+                    _('Bounding box'),
+                    form_layout.Div(
+                        form_layout.Field('north', wrapper_class='column-stretch-4'),
+                        css_class="box-column-container balanced",
+                    ),
+                    form_layout.Div(
+                        form_layout.Field('west', wrapper_class='column-stretch-4'),
+                        form_layout.Field('east', wrapper_class='column-stretch-4'),
+                        css_class="box-column-container separated",
+                    ),
+                    form_layout.Div(
+                        form_layout.Field('south', wrapper_class='column-stretch-4'),
+                        css_class="box-column-container balanced",
+                    ),
+                ),
+                css_class='form-group',
+                id='bbox-values',
+            ),
+            id=FormModeMixin.MODE_NEW,
+        )
+
+
+class ExcerptOrderFormCommonPartMixin(forms.Form):
+    is_public = forms.BooleanField(
+        label=_('Public'),
+        required=False,
+        help_text=_("Others will see name and bounding box of this excerpt. "
+                    "They won't see that you created it or when you did so."),
+        initial=True,
+    )
+    formats = forms.MultipleChoiceField(
+        label=_("GIS export formats:"),
+        choices=available_format_choices,
+        widget=forms.CheckboxSelectMultiple,
+        required=True,
+    )
+
+    def form_layout(self):
+        return form_layout.Div(
+            form_layout.Fieldset(
+                _('Visibility'),
+                'is_public',
+            ),
+            form_layout.Fieldset(
+                _('GIS export formats'),
+                'formats',
+            ),
+        )
+
+
+class ExcerptOrderFormPartExistingMixin(forms.Form):
+    existing_excerpts = forms.ChoiceField(
+        label=_('Excerpt'),
+        required=True,
+    )
+
+    def form_layout(self):
+        return form_layout.Div(
+            form_layout.HTML('<h2>' + _('Existing excerpt') + '</h2>'),
+            form_layout.HTML('''
+                <div class="form-group has-feedback">
+                    <input id="excerptListFilterField" class="form-control" type="search" placeholder="Filter excerpts …" autocomplete="off"/>
+                    <span id="excerptListFilterFieldClearer" class="clearer glyphicon glyphicon-remove-circle form-control-feedback"></span>
+                </div>
+                '''  # noqa: line too long ignored
```

Comment:
Should this long string (HTML snipped for the excerpt name filter text input field) be inline in this `form_layout` method or extracted to a class or module constant (or, with the wrapping call to `form_layout.HTML`, to a private method)?